airbyte-integrations/connectors/source-shopify/source_shopify/shopify_graphql/bulk/job.py

```diff
@@ -283,14 +325,19 @@ def _on_job_with_errors(self, errors: List[Mapping[str, Any]]) -> AirbyteTracedE
         raise ShopifyBulkExceptions.BulkJobError(f"Could not validate the status of the BULK Job `{self._job_id}`. Errors: {errors}.")
 
     def _on_non_handable_job_error(self, errors: List[Mapping[str, Any]]) -> AirbyteTracedException:
-        raise ShopifyBulkExceptions.BulkJobNonHandableError(f"The Stream: `{self.stream_name}`, Non-handable error occured: {errors}")
+        raise ShopifyBulkExceptions.BulkJobNonHandableError(f"The Stream: `{self.http_client._name}`, Non-handable error occured: {errors}")
+
+    def _get_server_errors(self, response: requests.Response) -> List[Optional[dict]]:
```

Comment:
nit:  for consistency can we make this `Mapping[str, Any]`. We'll also end up changing `_collect_bulk_errors()` signature, but that probably should be done anyway since the consuming method `_job_healthcheck` also uses a mapping

Reply:
Corrected! Thanks!